docs/api-reference/io-columns-multiclass-classification.md

```diff
@@ -6,4 +6,4 @@ This trainer outputs the following columns:
 | Output Column Name | Column Type | Description|
 | -- | -- | -- |
 | `Score` | vector of <xref:System.Single> | The scores of all classes. Higher value means higher probability to fall into the associated class. If the i-th element has the largest value, the predicted label index would be i. Note that i is zero-based index. |
```

Comment:
>vector [](start = 12, length = 6)

Shall we start sentences with capital letter?

Reply:
Spoke offline with @Ivanidzo4ka  @shmoradims and got feedback from @natke that either "key" or "Key" is fine.